src/packages/@ncigdc/utils/filters/index.js

```diff
@@ -232,4 +233,20 @@ export const makeFilter: TMakeFilter = (fields, returnString = true) => {
   return returnString ? JSON.stringify(filters) : filters;
 };
 
+type TRemoveFilter = (field: string, query: TUriQuery) => TUriQuery;
```

Comment:
hmm, I could make the removeFilter function return that information too? seems like it could be a common use case.